test/output-tests.js

```diff
@@ -28,8 +28,8 @@ Object.keys(files).forEach((absPath) => {
 describe("output", () => {
   function check(data) {
     const code = compile(data.actual).code;
-    // Trim blank lines and trailing whitespace.
-    const actual = code.replace(/^ +$/gm, "").trimRight();
+    // Consolidate semicolons then trim blank lines and trailing whitespace.
+    const actual = code.replace(/;{2,}/g, ";").replace(/^ +$/gm, "").trimRight();
```

Comment:
While I removed the rogue semicolon in acorn (due to the acorn-extensions) it still exists in babylon.